From: SZALAY Attila <sasa@debian.org>
Date: Thu, 2 Feb 2012 22:18:55 +0100
Subject: [PATCH] compile_problems

===================================================================
---
 lib/proxystack.c |    3 ---
 1 files changed, 0 insertions(+), 3 deletions(-)

diff --git a/lib/proxystack.c b/lib/proxystack.c
index 14c9ffb..5a2f67e 100644
--- a/lib/proxystack.c
+++ b/lib/proxystack.c
@@ -234,8 +234,6 @@ z_proxy_control_stream_read(ZStream *stream, GIOCondition cond G_GNUC_UNUSED, gp
   if (strcmp(request->command->str, "SETVERDICT") == 0
      )
     {
-      ZProxyStackIface *siface;
-      
       iface = z_proxy_find_iface(proxy, Z_CLASS(ZProxyStackIface));
       if (!iface)
         {
@@ -243,7 +241,6 @@ z_proxy_control_stream_read(ZStream *stream, GIOCondition cond G_GNUC_UNUSED, gp
           goto error;
         }
         
-      siface = (ZProxyBasicIface *) iface;
       if (strcmp(request->command->str, "SETVERDICT") == 0)
         {
           ZVerdict verdict;
-- 
