From: SZALAY Attila <sasa@ubainba.hu>
Date: Sat, 8 Oct 2016 17:50:03 +0200
Subject: Removed all variables related to the profession release

---
 configure.ac | 13 +------------
 1 file changed, 1 insertion(+), 12 deletions(-)

diff --git a/configure.ac b/configure.ac
index 0d6f779..b9bc253 100644
--- a/configure.ac
+++ b/configure.ac
@@ -3,7 +3,6 @@ dnl
 dnl There are a couple of environment defined variables which this script
 dnl makes use of in addition to the standard CXXFLAGS/LDFLAGS/etc. These are:
 dnl
-dnl RELEASE_TAG        - Debian release tag which is put to debian/changelog
 dnl
 
 AC_CONFIG_SRCDIR([zorp/main.cc])
@@ -15,7 +14,7 @@ dnl ***************************************************************************
 dnl Definitions
 
 BROCHURE_VERSION="7"
-ZORP_PRODUCT_NAME="Zorp Professional"
+ZORP_PRODUCT_NAME="Zorp GPL"
 
 dnl ***************************************************************************
 dnl Dependencies
@@ -45,9 +44,6 @@ case "$srcdir" in
 esac
 
 m4_ifdef([AM_SILENT_RULES],[AM_SILENT_RULES([yes])])
-if test -z "$RELEASE_TAG"; then
-        RELEASE_TAG=zorp35dbg
-fi
 
 AM_CONFIG_HEADER(lib/zorp/zorpconfig.h)
 AC_PREFIX_DEFAULT(/usr/local)
@@ -282,12 +278,8 @@ dnl ***************************************************************************
 dnl Size check of pointers
 AC_CHECK_SIZEOF(void *)
 
-AC_SUBST(RELEASE_TAG)
 AC_SUBST(BROCHURE_VERSION)
 
-AC_SUBST(ZORP_PACKAGE_LICENSE)
-AC_SUBST(PACKAGE_SUFFIX)
-
 AC_DEFINE_UNQUOTED(ZORP_PRODUCT_NAME, "$ZORP_PRODUCT_NAME", [Required product name in license])
 AC_SUBST(ZORP_PRODUCT_NAME)
 
@@ -374,7 +366,6 @@ ipoptions: $enable_ip_options
 ipv6: $enable_ipv6
 prefork: $enable_prefork
 kzorp: $enable_kzorp
-License: $ZORP_PACKAGE_LICENSE
 
 
 EOF
@@ -385,8 +376,6 @@ AC_SUBST(LIBZORP_LT_COMPAT_PL)
 
 AC_SUBST(GLIB_MIN_VERSION)
 AC_SUBST(ZORPLL_MIN_VERSION)
-AC_SUBST(ZORPLIC_MIN_VERSION)
-AC_SUBST(ZORPMISC_MIN_VERSION)
 AC_SUBST(OPENSSL_MIN_VERSION)
 AC_SUBST(PYTHON_MIN_VERSION)
 AC_SUBST(CURRDATE)
