From: SZALAY Attila <sasa@ubainba.hu>
Date: Thu, 21 May 2015 22:32:30 +0200
Subject: Did some more transformation to compile

---
 configure.ac |   30 ++++++++++++------------------
 1 file changed, 12 insertions(+), 18 deletions(-)

diff --git a/configure.ac b/configure.ac
index 8755c7a..6281977 100644
--- a/configure.ac
+++ b/configure.ac
@@ -8,23 +8,16 @@ dnl SNAPSHOT_VERSION   - snapshot version to add to version number
 dnl SOURCE_REVISION    - Revision of the source-tree, will added to the version string
 dnl
 
+AC_INIT([zorp], m4_esyscmd([tr -d '\n' <VERSION]))
+AC_CONFIG_SRCDIR(zorp/main.c)
+AM_INIT_AUTOMAKE([no-define foreign])
+m4_ifdef([AM_SILENT_RULES],[AM_SILENT_RULES])
+
 
 dnl ***************************************************************************
 dnl Definitions
 
-PACKAGE=zorp
 BROCHURE_VERSION="3.9"
-VERSION=`cat $srcdir/VERSION`
-ZORP_PRODUCT_NAME="Zorp Professional"
-ZORP_LICENSE_VERSION="3.4"
-
-dnl ***************************************************************************
-dnl Dependencies
-
-GLIB_MIN_VERSION="2.2.1"
-ZORPLL_MIN_VERSION="3.9.1.0"
-OPENSSL_MIN_VERSION="0.9.8"
-PYTHON_MIN_VERSION="2.6"
 
 dnl ***************************************************************************
 dnl Initial setup
@@ -46,10 +39,13 @@ if test -r $srcdir/dist.conf; then
         source $srcdir/dist.conf
 fi
 
-AC_INIT($PACKAGE, $VERSION)
-AC_CONFIG_SRCDIR(zorp/main.c)
-AM_INIT_AUTOMAKE([no-define foreign])
-m4_ifdef([AM_SILENT_RULES],[AM_SILENT_RULES])
+dnl ***************************************************************************
+dnl Dependencies
+
+GLIB_MIN_VERSION="2.2.1"
+ZORPLL_MIN_VERSION="3.9.1.0"
+OPENSSL_MIN_VERSION="0.9.8"
+PYTHON_MIN_VERSION="2.6"
 
 if test -n "$SNAPSHOT_VERSION"; then
         VERSION=$VERSION+$SNAPSHOT_VERSION
@@ -389,8 +385,6 @@ AC_SUBST(BROCHURE_VERSION)
 AC_SUBST(SNAPSHOT_VERSION)
 AC_SUBST(SOURCE_REVISION)
 
-AC_DEFINE_UNQUOTED(ZORP_PRODUCT_NAME, "$ZORP_PRODUCT_NAME", [Required product name in license])
-
 AC_DEFINE_UNQUOTED(BROCHURE_VERSION, "$BROCHURE_VERSION", [Zorp brochure version])
 AC_DEFINE_UNQUOTED(VERSION, "$VERSION", [Zorp package version])
 AC_DEFINE_UNQUOTED(ZORP_SOURCE_REVISION, "$SOURCE_REVISION", [Zorp source revision number])
