From: SZALAY Attila <sasa@ubainba.hu>
Date: Fri, 9 Dec 2016 13:18:46 +0100
Subject: Removed option which prevent building

the package
---
 libzorp.pc.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libzorp.pc.in b/libzorp.pc.in
index 3f5f9da..8dddfc6 100644
--- a/libzorp.pc.in
+++ b/libzorp.pc.in
@@ -9,5 +9,5 @@ Name: ZorpLib
 Description: Zorp Core Library
 Requires: zorplibll
 Version: @VERSION@
-Libs: -lzorp -L@libdir@ @PYTHON_LIBS@
-Cflags: -I@includedir@ @PYTHON_CFLAGS@
+Libs: -lzorp -L@libdir@
+Cflags: -I@includedir@
