From: SZALAY Attila <sasa@ubainba.hu>
Date: Tue, 26 May 2015 21:37:14 +0200
Subject: Removed test subdir because of missing files

---
 modules/ftp/Makefile.am |    2 --
 1 file changed, 2 deletions(-)

diff --git a/modules/ftp/Makefile.am b/modules/ftp/Makefile.am
index 79cb316..98ffc5b 100644
--- a/modules/ftp/Makefile.am
+++ b/modules/ftp/Makefile.am
@@ -1,5 +1,3 @@
-SUBDIRS = tests
-
 pkgdatadir = @datadir@/zorp/pylib/Zorp
 pkglibdir = @libdir@/zorp
 LIBS = @MODULES_LIBS@
