From: SZALAY Attila <sasa@ubainba.hu>
Date: Tue, 26 May 2015 21:49:27 +0200
Subject: tests/Makefile.am: Removed kzorp directory too because of an error

---
 tests/Makefile.am |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/Makefile.am b/tests/Makefile.am
index adc6ec4..ff6d048 100644
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -1,4 +1,4 @@
 
-SUBDIRS = unit kzorp
+SUBDIRS = unit
 
 EXTRA_DIST = functional/README.txt @TESTCASES@
