From: =?utf-8?q?Jakab_Krist=C3=B3f?= <jakab.kristof@balasys.hu>
Date: Thu, 5 Jan 2017 14:48:12 +0100
Subject: pydict.cc: handle IPv4 addresses as unsigned numbers

References: jira ZORP-869
---
 lib/pydict.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/pydict.cc b/lib/pydict.cc
index 977e73f..49a5e8a 100644
--- a/lib/pydict.cc
+++ b/lib/pydict.cc
@@ -784,7 +784,7 @@ z_policy_dict_ip_get_value(ZPolicyDict *self G_GNUC_UNUSED, ZPolicyDictEntry *en
     }
   else if (entry->type == Z_VT_IP)
     {
-      res = PyInt_FromLong(((struct in_addr *) entry->value)->s_addr);
+      res = PyLong_FromUnsignedLong(((struct in_addr *) entry->value)->s_addr);
     }
   else
     {
