From: SZALAY Attila <sasa@ubainba.hu>
Date: Tue, 26 May 2015 21:54:57 +0200
Subject: Makefile.am: Removed test directory because of an erro rin
 compilation

---
 Makefile.am |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 22c0c3d..2282a5a 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,4 +1,4 @@
-SUBDIRS = lib libproxy modules zorp pylib doc zorpctl scripts tests
+SUBDIRS = lib libproxy modules zorp pylib doc zorpctl scripts
 DIST_SUBDIRS = $(SUBDIRS)
 
 pkgconfigdir = $(libdir)/pkgconfig
