From: =?utf-8?q?Szil=C3=A1rd_Pfeiffer?= <szilard.pfeiffer@balasys.hu>
Date: Wed, 1 Feb 2017 10:30:23 +0100
Subject: all: Fix compilation error with GCC 7

closes #10
---
 modules/http/http.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules/http/http.cc b/modules/http/http.cc
index 8dc9580..01a4a5b 100644
--- a/modules/http/http.cc
+++ b/modules/http/http.cc
@@ -44,6 +44,7 @@
 #include <fcntl.h>
 
 #include <memory>
+#include <functional>
 
 #include <netdb.h>
 static GHashTable *auth_hash = NULL;
